brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-2229 Fix: Post Grid - Undefined array key issue.

### Description

- https://brainstormforce.slack.com/files/U03DUKFKEKC/F04HJUV7SKB/image.png

### Screenshots

- Issue - https://share.getcloudapp.com/BluRx8Ab
- Testing evidence - https://share.getcloudapp.com/xQuO6XXy

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

Test case  1
- Drag and drop the Post grid in v1.25.6
- Publish and visit frontend.
- Now switch to master, and refresh the frontend after regenerating assets you will get that issue.
- Now switch to a mentioned branch, regenerate the assets refresh the frontend.
- Verify you will not get that issue again.

Test Cases 2
- Drag and drop Post Grid.
- Verify whether buttons are inline properly or not.
- Option available in Read More > Buttons Equal Height.
- Change the image position to the background from Image > Image Position > Background.
- Verify whether buttons are inline or not.

Screencast of both - https://share.getcloudapp.com/xQuO6XXy

### Checklist:
